src/pages/NotFoundPage.tsx

```diff
@@ -0,0 +1,5 @@
+import { FC } from 'react';
+
+export const NotFoundPage: FC = () => (
+  <h1 className="title">Page not found</h1>
+);
```

Comment:
Do you need 3 same components?

You can reduce amount of code by creating component Page and pass the title inside it

`<Page title={"Home page"}/>`

![image](https://github.com/mate-academy/react_tabs-with-router/assets/66055069/7938a360-c43e-496e-930e-9d475fba2a63)

Reply:
@ivankhrsh it can be a good idea, but these components can increase. In my opinion, it is a good practice to create separate files